### DISCUSSION TITLE
0-100kmh logging

### DISCUSSION THREAD

**Discussion by tb205gti** [teslamate-org/teslamate#807] [Ideas]
**What would you like to be added**:

A table showing the 0-100kmh time if any is present in a drive. Perhaps also 0-60kmh and 0-130kmh. Table should include SOC and max KW. The table could be part of the drive details dashboard

**Why is this needed**:
Because it is interesting, the streaming API is quick enough for the data. :)

**Additional context**:

**Comment by Dulanic** [comment]
I don't think it would be very valid. Thats like using a 1hz GPS 0-60 app... not really useful when your accuracy is 1 second and your measure 3.2, 3.8, 4.3 etc....

FYI, it isn't a BAD idea, but I've tried to capture that myself before from the raw data and it just wasn't fine enough detail to be useful.

**Comment by tb205gti** [comment]
Well it is ok fine resolution - not proper equipment precise, but reasonably good. I get data at 250 ms intervals at worst - and 75 ms at best.

**Comment by tb205gti** [comment]
one thing I'd like to see if I can find a trend for - as how much a low SOC affects the 0-100 time. It has a massive effect on the power - but how does that translate to 0-100. (I know it will be lower..)
